### ISSUE TITLE
Keyboard accessibility

### ISSUE THREAD

**Issue by da2x** [jaesivsm/JARR#64]
I’m missing three primary interactions in terms of keyboard accessibility.

1. Arrow keys Up and Down should take me to the previous and next items in the list of new fee entries/clusters. (When the list has focus.)
2 . Ctrl+Enter should open the currently open feed entry.
3. The list of categories is never focused when Tab cycling. Also, no up/down key navigation to change selection.

**Comment by blatinier** [comment]
This is an appendix of #36 I think.
